target/linux/ath79/dts/qca9531_yuncore_a770.dts

```diff
@@ -123,6 +127,13 @@
 
 &pcie0 {
 	status = "okay";
+
+	wifi@0,0 {
+		compatible = "qcom,ath10k";
+		reg = <0x0000 0 0 0 0>;
+		nvmem-cells = <&cal_art_5000>;
+		nvmem-cell-names = "calibration";
```

Comment:
here we used calibration but in the other case we use pre-calibration? Think we have to use calibration in these strange case to remove any chance of causing regression...